sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/DebugTest.groovy

```diff
@@ -0,0 +1,23 @@
+package com.azure.storage.blob
+
+import com.azure.core.util.serializer.JacksonAdapter
+import com.azure.core.util.serializer.SerializerEncoding
+import com.azure.storage.blob.models.BlobServiceStatistics
+import spock.lang.Specification
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.class DebugTest
+
+class DebugTest extends Specification {
```

Comment:
should this go to one of the core test suites ?

Reply:
I'm planning on deleting this, didn't mean for it to be included in the commit.